src/StateSmith/Output/Sim/HtmlRenderer.cs

```diff
@@ -136,15 +190,24 @@ public static void Render(StringBuilder stringBuilder, string smName, string moc
 
     <div class=""pane sidebar"">
         <div id=""buttons"">
-            <div class=""titlebar"">Events</div>
+            <div class=""titlebar"">Events            
+              <div class='dropdown'>
+                <span id='dropbtn' class='titlebar-icon dropbtn'>settings</span>
+                <div id='myDropdown' class='dropdown-content'>
+                  <div class='dropdown-item'>
+                    <input type='checkbox' id='timestamps' name='timestamps' value='Timestamps'>
+                    <label for=timestamps'>Timestamps</label>
```

Comment:
```suggestion
                    <label for='timestamps'>Timestamps</label>
```
The settings dropdown is great! I played around with enabling/disabling the timestamps and noticed clicking the label didn't seem to work. Looks like missing a single quote.

Reply:
@emmby I tried applying a suggestion to fix typo, but realized that the code gen should be run again anyways. Can you pull, rerun code gen and then push?

Everything else looks great!

Reply:
done!